pages/products.html

```diff
@@ -0,0 +1,308 @@
+          <span class="material-icons"> search </span>
+        </div>
+      </div>
+      <div class="nav-items">
+        <a class="black-text-color"  href="./wishlist.html">
+          <span class="material-icons mr-1" title="wishlist"> favorite </span>
+        </a>
+        <a class="black-text-color"  href="./cart.html">
+          <span class="material-icons mr-1 ml-1" title="cart">
+            shopping_cart
+          </span>
+        </a>
+        <a  href="./login.html" class="black-text-color"
+          ><span class="material-icons ml-1" title="account">
+            account_circle
+          </span>
+        </a>
+      </div>
+    </div>
+    <div class="d-flex">
+     <div class="d-flex flex-column sidebar-container">
+        <div class="d-flex flex-justify-space-between mb-2">
+           <h4>Filters</h4> 
+           <a class="underline">Clear</a>
+        </div>
+        <div class="mb-2">
+            <h4 class="pb-1">Price</h4>
+            <input type="range" name="range"  min="0" max="100" value="50"/>     
+        </div>
+        <div class="mb-2">
+            <h4 class="pb-1">Category</h4>
+            <ul class="list-style-none">
+                <li>
+                    <input id="checkbox-checked" class="form-check-input" type="checkbox" checked/>
+                   <label for="checkbox-checked">Furniture</label>
+                </li>
+                <li>
+                    <input id="checkbox-unchecked" class="form-check-input" type="checkbox"/> 
+                    <label for="checkbox-unchecked">Kitchen & appliances</label>
+                </li>
+                <li>
+                    <input id="checkbox-unchecked" class="form-check-input" type="checkbox"/> 
+                    <label for="checkbox-unchecked">Beds & mattresses</label>
+                </li>
+                <li>
+                    <input id="checkbox-unchecked" class="form-check-input" type="checkbox"/> 
+                    <label for="checkbox-unchecked">Decoration</label>
+                </li>
+            </ul>
+         </div>
+         <div class="mb-2">
+            <h4 class="pb-1">Rating</h4>
+            <ul class="list-style-none">
+                <li>
+                    <input id="radio-checked" class="form-check-input" type="radio" checked/>           
+                    <label for="radio-checked">4 stars & above</label>
+          
+                </li>
+                <li>
+                    <input id="radio-unchecked" class="form-check-input" type="radio"/>
+                    <label for="radio-unchecked">3 stars & above</label>
+                </li>
+                <li>
+                    <input id="radio-unchecked" class="form-check-input" type="radio"/>
+                    <label for="radio-unchecked">2 stars & above</label>
+                </li>
+                <li>
+                    <input id="radio-unchecked" class="form-check-input" type="radio"/>
```

Comment:
I saw the preview, it looks awesome, just a suggestion, during using input type radio for filter purpose  give them name the same name so that at a time only one get selected  !

Reply:
Really insightful suggestion ! added the name attribute.